economics.toml

```diff
@@ -24,24 +24,22 @@
     LeaderPercentage = 0.1 #fraction of value 0.1 - 10%
     DeveloperPercentage = 0.3 #fraction of value 0.3 - 30%
     ProtocolSustainabilityPercentage = 0.1 #fraction of value 0.1 - 10%
-    ProtocolSustainabilityAddress = "erd135enn78u7fy352ny9mcl26a08wvu02mjdumfkt7cn587cwtnerksagqsfm"
+    ProtocolSustainabilityAddress = "erd1j25xk97yf820rgdp3mj5scavhjkn6tjyn0t63pmv5qyjj7wxlcfqqe2rw5"
     TopUpGradientPoint = "3000000000000000000000000" # 3MIL eGLD (eligible topUp)
     TopUpFactor = 0.25 # fraction of value 0.25 - 25%
 
     [[RewardsSettings.RewardsConfigByEpoch]]
```

Comment:
Maybe keep only one `RewardsConfigByEpoch`

Reply:
> 1.Maybe keep only one RewardsConfigByEpoch

It's ok, it's better to have more to showcase how to add them


> 2.Is it needed to lower the rewards for validators?

Economically, yes

Reply:
Still don't think is needed to showcase examples for `RewardsConfigByEpoch`, we should only have epoch 0.